### PR TITLE
Pull request from features to master

### PR DESCRIPTION
**Pull request by phalado**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    height: 100%;
+    text-align: center;
+    align-items: center;
+}
+
+nav li{
+    display: table-cell;
+    width: 100%;
+
+    /*border-color: black;*/
```
Comment (Redvanisation) [inline css/styles.css]: Delete commented code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                <li>
+                    <form class="list search" action="#">
+                        <input type="search" name="search">
+                        <div class="magnify"></div>
+                    </form>
+                </li>
+            </ul>
+        </nav>
+        <main class="big-picture">
+            <a href="/web/20140301004610/http://www.apple.com/your-verse/">
```
Comment (Redvanisation) [inline index.html]: Only the titles should be links and not the whole background image

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @phalado   🌟

**Great work on this project, but we need to fix the issues below before moving on ⬇️** 

1. Remove the white margin between the four bottom elements.

2. Both of the footer elements should be aligned like on the original website.

3. Please fix the things I commented on below.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: index.html (hunk 2 above)
Comment (phalado) [reply index.html]: I've already corrected it, but in the original page, the whole background image is a link.

**Review by suretrust — APPROVED** [review]
Amazing job @phalado, :raised_hands: 

### Looks good to me, you can go ahead and merge! :tada: 

_Good luck with your future projects and happy coding._

**Pull request merged**